scipy/linalg/_decomp_cholesky.py

```diff
@@ -218,7 +218,12 @@ def cho_solve(c_and_lower, b, overwrite_b=False, check_finite=True):
     True
 
     """
-    (c, lower) = c_and_lower
+    c, lower = c_and_lower
+    return _cho_solve(c, lower, b, overwrite_b=overwrite_b, check_finite=check_finite)
+
+
+@_apply_over_batch(('c', 2), ('lower', 0), ('b', '1|2'))
```

Comment:
`cho_factor` doesn't support batching for `lower` so it seems maybe a little asymmetric to have it here

Reply:
Removed.